testapps/testapp/src/main/java/com/microsoft/identity/client/testapp/AcquireTokenFragment.java

```diff
@@ -599,6 +602,7 @@ private RequestOptions getCurrentRequestOptions() {
         final String claims = mClaims.getText().toString();
         final boolean enablePII = mEnablePII.isChecked();
         final boolean forceRefresh = mForceRefresh.isChecked();
+        final boolean allowSignInFromOtherDevice = mAllowSignInFromOtherDevice.isChecked();
```

Comment:
just curious, do we really need this option? I mean, what is the harm in always having to pass the is_remote_login_allowed flag as true? It won't be exercised unless user clicks on the "sign in from another device" button in ests right?

Reply:
I don't think there's a way currently in msal test app to pass withAuthorizationQueryStringParameters() to the AcquireTokenParameters object. I think having it as an option makes sense, because we may want to test without it, instead of always passing it in.

Reply:
We should also test with not passing it, that's why. I should a UI test for that as well, I will update the other automation pr with it. "Sign in from other device" should not be there, if this flag is not passed.